csrc/transform_replay.cpp

```diff
@@ -300,7 +300,8 @@ std::pair<TensorDomain*, size_t> TransformReplay::replayPasC(
     int64_t consumer_pos,
     const RootDomainMap& root_map,
     bool replay_swizzle,
-    bool replay_resize) {
+    bool replay_resize,
+    bool replay_allocation) {
```

Comment:
End up not needing `replay_allocation` in `replayPasC`, but I believe it is good to keep it to maintain symmetry.